core/src/lib.rs

```diff
@@ -6,3 +10,58 @@ mod http;
 pub mod release;
 pub mod storage;
 pub mod tracker;
+
+/// The different games supported by the launcher
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum Game {
+    Jumpy,
+    Punchy,
+}
+
+#[derive(thiserror::Error, Debug)]
+#[error("Invalid game ID")]
+pub struct InvalidGameId;
```

Comment:
This can be a part of `Error` struct. There is no need to define another type for error handling.